docs/install/_linux.rst

```diff
@@ -42,7 +42,7 @@ Upgrade
 When you use the PPA installation method, upgrades to newer versions will be automatic, provided there is internet access available.
 
 kolibri-server (pre-release)
-****************************
+----------------------------
```

Comment:
@jredrejo what would you say if we removed "(pre-release)"? Thinking that the latest feedback has been encouraging. You still need the cache stuff to go in?

Reply:
Not necessarily, I'm good with removing this if you agree, will amend asap! :+1:

Reply:
Yeah, I would also defend that decision, simply to get a gradually wider adaptation and keeping the huge benefits of this package in mind.